### PR TITLE
update deps to support node v0.12.x

### PR DESCRIPTION
**Pull request by flosse**
I cannot install the latest stable with node `v0.12.x` because of its dependencies. Updating the deps would help. Can we create a `v0.4.x` branch and publish a `v0.4.11` version?
(there is also a `v0.5` on npmjs but the git repo has no corresponding tag).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-xmpp-core",
-  "version": "0.4.10",
+  "version": "0.4.11",
```
Comment (chris-rock) [inline package.json]: can we remove the version bump? we should do this seperately

**Comment by chris-rock** [comment]
@flosse thank you very much! @lloydwatkin this looks good after the minor change

File: package.json (hunk 1 above)
Comment (lloydwatkin) [inline package.json]: Yeah version bump will be done when we tag :)

**Comment by lloydwatkin** [comment]
@flosse can you merge in master and then we can get this merged in. Thanks!

**Comment by flosse** [comment]
> can we remove the version bump? we should do this seperately

yes of course

> can you merge in master

no, that's not the idea. I'd like to have a stable branch that is working with node `v0.10` and `v0.12`
So publishing a stable `v04.11` based on a alpha master branch doesn't make sense so me.
Instead I'd create a `v0.4.x` branch separately.

**Comment by sonnyp** [comment]
fixed

**Pull request closed without merging**